packages/grid/src/types/index.ts

```diff
@@ -23,7 +23,7 @@ export type TextAlign = typeof TEXT_ALIGN[number];
 export type GridNumber = number | string;
 export type Breakpoint = number | string | boolean;
 export type Space = typeof SPACE[number];
-export type Wrap = 'nowrap' | 'wrap' | 'wrap-reverse';
+export type Wrap = typeof WRAP[number];
```

Comment:
Discovered this oversight while reviewing https://github.com/zendeskgarden/website/pull/375